extensions/formats/gpx/src/main/java/mil/nga/giat/geowave/format/gpx/GpxIngestFormat.java

```diff
@@ -22,10 +26,14 @@
 public class GpxIngestFormat extends
 		AbstractSimpleFeatureIngestFormat<GpxTrack>
 {
+	private final GeometrySimpOptionProvider simplifyOptProvider = new GeometrySimpOptionProvider();
```

Comment:
I thought you intended to put this in `AbstractSimpleFeatureIngestFormat` so it was more broadly available to other formats?